### DISCUSSION TITLE
So.. My Sender Going Straight to Spam - Suggetions?

### DISCUSSION THREAD

**Discussion by DanielleLynn** [merchantprotocol/shadowmailer-support#26] [Open Forum]
https://www.mail-tester.com/test-uzqfhalsr

Hey guys - so I'm about to do a broadcast.  I ran a quick test and saw my message is gonna go straight to spam.  

I was thinking that the 'warming' was going to help ensure the messages would be sent out. 

Any quick tips on what I can do here to improve this?

**Comment by jonathonbyrdziak** [comment]
This is a dns issue. I'll have your account manager fix it

**Reply by DanielleLynn** [reply]
Awesome - can I get a heads up when fixed?

I'm also having a challenge getting the domain tracker to work > https://github.com/merchantprotocol/shadowmailer-support/issues/19

**Reply by DanielleLynn** [reply]
<img width="693" alt="Screen Shot 2022-08-30 at 3 12 26 PM" src="https://user-images.githubusercontent.com/94147553/187552849-2a24653c-4c42-4a88-a4ee-8b6fc3180905.png">


I'm curious if there's anything that we can do to fix this - currently I've gotten it to a 6/10
